### ISSUE TITLE
create package as-types

### ISSUE THREAD

**Issue by gregLibert** [massalabs/as#40]
The idea is to transform massa/as to a group a multiple independant repos.

We should create as-type and migrate there:
- amount
- currency
- result

**Comment by gregLibert** [comment]
Should be started after #38 to reduce conflict

**Comment by peterjah** [comment]
Should be started after https://github.com/massalabs/as/issues/53

**Comment by Thykof** [comment]
Task specification update: now that we created `as-proba` package inside `as` monorepo, this task is to:
- [ ] rename the package `as` into `as-types`
- [ ] mark `@massalabs/as` is deprecated in npm js
- [ ] create tasks to update repositories that depend on the deprecated package

**Comment by Thykof** [comment]
Follow up issues:
- https://github.com/massalabs/massa-as-sdk/issues/113
- https://github.com/massalabs/massa-sc-toolkit/issues/71
- https://github.com/massalabs/massa-sc-contracts/issues/34

**Comment by Thykof** [comment]
# QA 

package as-types published here: https://www.npmjs.com/package/@massalabs/as-types

**Comment by longgamma-massa** [comment]
Let's add a Readme, at least a short one. And with a new issue we will work on the better Readme's for all packages (soon). @Thykof

**Comment by Thykof** [comment]
@longgamma-massa issue for readme https://github.com/massalabs/as/issues/66
